src/Microsoft.Security.Utilities.Core/Marvin.NetFramework.cs

```diff
@@ -112,32 +112,32 @@ public static long ComputeHash(byte[] data, ulong seed, int offset, int length)
                 switch (length)
                 {
                     case 0:
-                    p0 += 0x80u;
-                    break;
+                        p0 += 0x80u;
```

Comment:
>+= [](http://example.com/codeflow?start=27&length=2)

What *should* have fired here is that we require case statements to be braced. I think that's why we don't require indentation.

This is what we want.

                    case 0:
                    {
                        p0 += 0x80u;
                        break;
                     }

Reply:
Ah, I see. I can't find a way to enforce braces, but I did find a way to make it format sensibly (IMHO) with or without braces. 

See latest changes.